cmd/docs.go

```diff
@@ -68,7 +68,7 @@ var docsCmd = &cobra.Command{
 				u.LogErrorAndExit(schema.AtmosConfiguration{}, fmt.Errorf("Component '%s' not found in path: '%s'", info.Component, componentPath))
 			}
 
-			readmePath := filepath.Join(componentPath, "README.md")
+			readmePath := path.Join(componentPath, "README.md")
```

Comment:
This PR is reverting a change from `path.Join` to → `filepath.Join` due to inadequate testing. We will open a new PR to migrate from `path.Join` to → `filepath.Join`, so that we do not break `main`.